src/AutoscaleManager/NodeManager/NodeManagerService.cs

```diff
@@ -38,19 +39,31 @@ protected override async Task RunAsync(CancellationToken cancellationToken)
 
             while (!cancellationToken.IsCancellationRequested)
             {
-                try
+                if (string.IsNullOrEmpty(this.nodeManagerSettings.NodeTypesToManage))
                 {
-                    await RemoveScaledInNodesAsync(cancellationToken);
+                    Context.CodePackageActivationContext.ReportApplicationHealth(
+                       new HealthInformation("NodeManager", "NodeManagerSettings", HealthState.Warning)
+                       {
+                           RemoveWhenExpired = true,
+                           TimeToLive = this.nodeManagerSettings.ScanInterval,
+                           Description = "Add atleast one NodeType in NodeTypestoManage in AutoScaleHelper Service Manifest."

```

Comment:
Required Parameter NodeTypesToManage is empty. Set this parameter to configure the comma separated nodetype name list, to be managed by the AutoScaleHelper.